bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -242,3 +242,6 @@ decimal.value.out.of.range = decimal range overflow
 config.size.mismatch = [{0}] the size for configurable variable ''{1}'' is expected to be ''{2}'', \
   but found ''{3}''
 invalid.utf8.byte.array.value = array contains invalid UTF-8 byte value
+negative.regexp.match.string.index = start index cannot be less than 0
+regexp.match.string.index.out.of.range = start index ''{0}'' cannot be greater than input length ''{1}''
+regexp.invalid.pattern = invalid regexp pattern
```

Comment:
I think you have missed to add "regexp.empty.string" and "regexp.empty" values.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39444/commits/e1ae14d4aecebf28ef562a2b13824f852931b87c.